### DISCUSSION TITLE
nvs.bin file makes every changed button to output1.

### DISCUSSION THREAD

**Discussion by alexgame8818** [darthcloud/BlueRetro#939] [Q&A]
hi~ always thank you for your Blue project.
I'm very exciting that nvs.bin file can save the button configration from firmware v1.9.2.
But after generating the bin file and flashed to ESP32,
I found device1 is OK ，but from device2 every changed button shows output1.
So from device2,I have to put every changed button  from output1 to their right output number by Web advance config.
Is there anything I can do  to solve the problem during the process of bin file generationg? thank you :)
(I tried NES and PCE.)

**Comment by darthcloud** [comment]
Ha yeah an oversight, its a bug, I will fix in next release.

**Reply by alexgame8818** [reply]
It really helps so much. 
Look forward to the next update coming. 
Thanks so much again :)

**Comment by darthcloud** [comment]
Should be fixed in this beta:
https://github.com/darthcloud/BlueRetro/actions/runs/8609471801

**Reply by alexgame8818** [reply]
It really works！I tried SNES and NES, and both are OK.
Another very fantastic job~
It's so flexible to switch among different individual keymaps.
Thank you so much let us can enjoy it before next update.
I will try more :}
